src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,46 @@
+package duke.main;
```

Comment:
Hey I like how your classes were split, made it very clear when looking for links between code! 👍 
I think JavaDocs should be added to some of the important methods and classes for clarity!